setup.py

```diff
@@ -51,6 +51,11 @@
             "face-mask-detector=face_mask_detector.command_line_interface:main"
         ]
     },
+    classifiers=[
```

Comment:
This modification will raise an error because your list is not comma separated.

Please delimit each list element with a comma.